utils/regression-analysis/src/main.rs

```diff
@@ -132,7 +128,7 @@ fn collect_data<P: AsRef<Path>>(
 
                     time_vec.insert(i, time);
                 } else {
-                    let mut time_vec = Vec::with_capacity(preallocate);
+                    let mut time_vec = Vec::with_capacity(PREALLOCATE);
```

Comment:
The intention of the argument was to not violate SRP however this doesn't matter too much for such auxiliary tool